pkg/git/repository.go

```diff
@@ -63,9 +64,11 @@ type (
 		url              string
```

Comment:
lets separate the public and private props with one blank line, just so it's easier to understand.